### ISSUE TITLE
Errors/exceptions when saving table structure changes

### ISSUE THREAD

**Issue by ghost** [sequelpro/sequelpro#691]
Issue imported from Google Code: https://code.google.com/p/sequel-pro/issues/detail?id=691
Reporter:glynnjwi...@googlemail.com Date:2010-05-18 05:05:52 Status:Fixed Closed:2010-09-01

What steps will reproduce the problem? Anything!

What version of Sequel Pro are you using? The Latest

What version of MySQL are you connecting to on the server? 5

I'm using an Apple MAC 1.6GHz PowerPC G5 running 10.5.9, with 3GB of RAM.

<b>Please provide any additional information below.</b>

Basically copying from one field to another, clicking onto a different
table, just about anything really.

I get:

&quot;NSRangeException
**\* -[NSCFArray objectAtIndex:]: index (10) beyond bounds (4)&quot;

I've sent the error to you every time. Have you seen it&gt; Do you know what
is causing it?

Kindest regards
Glynn
